dashboard/views.py

```diff
@@ -6,7 +7,19 @@
 from push.models import PushApplication
 
 
-class Home(TemplateView):
+class Login(TemplateView):
+    template_name = 'dashboard/login.html'
+
+
+class Status(TemplateView):
```

Comment:
Hmm ... I think we'd like to keep this on the home page? Or change this to a new `/push` url for a push home page.